models/project/src/mc-project/applied-permission/generator.ts

```diff
@@ -0,0 +1,12 @@
+import { fake, Generator } from '@commercetools-test-data/core';
+import { TAppliedPermission } from './types';
+
+const generator = Generator<TAppliedPermission>({
+  fields: {
+    // TODO: use more realistic values for action rights
+    name: fake((f) => f.animal.type()),
+    value: fake(() => true),
+  },
+});
```

Comment:
We can use real permissions names like this:

```suggestion
const fakePermissions = [
  'canViewProducts',
  'canManageProductTypes',
  'canViewStates',
  'canManageStates',
  'canViewOrders',
  'canManageOrders',
  'canAddOrders',
  'canAddDiscountCodes',
  'canManageProjectSettings',
];

const generator = Generator<TAppliedPermission>({
  fields: {
    name: fake((f) => f.helpers.arrayElement(fakePermissions)),
    value: fake(() => true),
  },
});```